src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImpl.java

```diff
@@ -224,17 +224,31 @@ private boolean validateAccountingLine(ConcurStandardAccountingExtractDetailLine
     private ConcurAccountInfo buildOverriddenConcurAccountingInformation(ConcurStandardAccountingExtractDetailLine line) {
         String overriddenObjectCode = getParameterService().getParameterValueAsString(CUKFSConstants.ParameterNamespaces.CONCUR, 
                 CUKFSParameterKeyConstants.ALL_COMPONENTS, ConcurParameterConstants.CONCUR_SAE_PDP_DEFAULT_OBJECT_CODE);
-        ConcurAccountInfo overriddenConcurAccountingInformation = new ConcurAccountInfo(line.getChartOfAccountsCode(), line.getAccountNumber(), 
-                line.getSubAccountNumber(), overriddenObjectCode, StringUtils.EMPTY, line.getProjectCode());
-        return overriddenConcurAccountingInformation;
+        return buildConcurAccountingInformation(line, overriddenObjectCode, StringUtils.EMPTY);
     }
 
     private void logErrorsWithOriginalAccountingDetails(ConcurStandardAccountingExtractDetailLine line) {
-        ConcurAccountInfo accountingInformation = new ConcurAccountInfo(line.getChartOfAccountsCode(), line.getAccountNumber(), 
-                line.getSubAccountNumber(), line.getJournalAccountCode(), line.getSubObjectCode(), line.getProjectCode());
+        ConcurAccountInfo accountingInformation = buildConcurAccountingInformation(
+                line, line.getJournalAccountCode(), line.getSubObjectCode());
         buildValidationResult(accountingInformation, false);
     }
     
+    private ConcurAccountInfo buildConcurAccountingInformation(
+            ConcurStandardAccountingExtractDetailLine line, String objectCode, String subObjectCode) {
+        if (lineRepresentsPersonalExpenseChargedToCorporateCard(line)) {
+            return new ConcurAccountInfo(line.getReportChartOfAccountsCode(), line.getReportAccountNumber(), 
+                    line.getReportSubAccountNumber(), objectCode, subObjectCode, line.getReportProjectCode());
+        } else {
+            return new ConcurAccountInfo(line.getChartOfAccountsCode(), line.getAccountNumber(), 
+                    line.getSubAccountNumber(), objectCode, subObjectCode, line.getProjectCode());
+        }
+    }
+    
+    private boolean lineRepresentsPersonalExpenseChargedToCorporateCard(ConcurStandardAccountingExtractDetailLine line) {
```

Comment:
I need this logic in additional places.  Can this bad added to the service contract.